### PR TITLE
See Issue #760 - Interview Information Sheet bug fix

### PR DESCRIPTION
**Pull request by levesswe**
Fix for the Format Currency error message.  #760

### CODE REVIEW

File: actions/interview-information-sheet.vbs [hunk 1]
```diff
@@ -575,8 +576,8 @@ CLASS doc_info
 
 	' Last payment date
 	PUBLIC PROPERTY GET last_payment_date
-		EMReadScreen at_screen, 4, 21, 75
-		IF at_screen <> "PALC" THEN CALL navigate_to_PRISM_screen("PALC")
+		EMReadScreen at_screen, 20, 2, 29
```
Comment (theVKC) [inline actions/interview-information-sheet.vbs]: @levesswe: from the surface, this doesn't appear to be related to the bug that you reported. If it is, can you tell me how? I presume it's a change to how it determines the current screen, but it's important to be clear in our commits what we're changing and why.

File: actions/interview-information-sheet.vbs [hunk 2]
```diff
@@ -779,7 +779,7 @@ IF participant = "NCP" Then
 			.FormFields("other_cases").Result = info.number_of_ncps_open_cases		
 			.FormFields("function").Result = info.case_function
 			.FormFields("assigned_worker").Result = info.worker_name & ", Phone: " & info.worker_phone
-			.FormFields("last_payment").Result = info.last_payment_date & " " & FormatCurrency(info.last_payment_allocation) & " " & info.last_payment_type
+			.FormFields("last_payment").Result = info.last_payment_date & " $" & FormatCurrency(CCur(info.last_payment_allocation)) & " " & info.last_payment_type
```
Comment (theVKC) [inline actions/interview-information-sheet.vbs]: This looks great. Would you be interested in writing out a brief description of why you had to change this, and sending it my way (via email is fine)? I might want to use this in a future training because it's a good function to learn. :)

File: actions/interview-information-sheet.vbs (hunk 1 above)
Comment (levesswe) [inline actions/interview-information-sheet.vbs]: In short, the FormatCurrency bug was cause by a failure of the script to navigate to the proper screen in order to obtain the requested information.  I had initially had the script look for "PALC" in the bottom right-hand corner of the screen, and display the Payment List By Case if it did not equal PALC.  However, we had a problem if PRISM displayed an individual payment record from PALC to display the Case/Payment Distribution List.  The bottom right-hand corner of the page still says "PALC", but we are not on the Payment List By Case screen that we need to be in for the script to read the information we are asking it to read.  Consequently, the script would read information that was not numeric, and therefore could not convert to dollar amounts.  Hope that helps!

Comment (theVKC) [inline actions/interview-information-sheet.vbs]: It does. Thank you!
